lib/gingr/cli.rb

```diff
@@ -13,13 +13,19 @@ class Cli < Thor
 
     Thor.check_unknown_options!
 
+    class << self
```

Comment:
We don't have a style checker on this repo yet, but in general we opt to keep all class methods in a `class << self` block. This ensures they're all kept together and you don't accidentally miss a `self.foo` declaration later in the file.